documentation/modules/ostack-token-auth.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="openstack-token-authentication_{context}"]
+= Using token authentication with an {osp} source provider
+
+You can use token authentication instead of username and password authentication when you use {project-first} to migrate a virtual machine that has an {osp} source provider.
```

Comment:
`migrate a virtual machine that has an {osp} source provider` sounds confusing, how about
```suggestion
You can use token authentication instead of username and password authentication when creating an {osp} source provider.
```

Reply:
ah I see - it appears twice and you've changed it here but not on the other place, commented there